release/_promotion.sh

```diff
@@ -44,6 +46,34 @@ function prepare_poms_for_promotion {
 	sed -i "s#<version>${_ARTIFACT_RC_VERSION}</version>#<version>${_PRODUCT_VERSION}</version>#" ./*.pom
 }
 
+function promote_boms {
+	lc_time_run prepare_poms_for_promotion ${1}
+
+	lc_time_run prepare_jars_for_promotion ${1}
```

Comment:
@lucasmiranda0, this change isn't only moving the code, actually it's also changing the usage of the functions above, e.g. using `${1}` instead `xanadu`. Can you please perform this change in a different commit?